### PR TITLE
Homework27: Ekaterina Tsiunel

### PR DESCRIPTION
**Pull request by ETsiunel**
invite link to the updated collection in Postman https://app.getpostman.com/join-team?invite_code=356e63a239a08691f83afc230cd3a60d&target_code=3951619ccb588b24113e1dd9a6b346fe

### CODE REVIEW

**Comment by ETsiunel** [comment]
**(!) Note:** 
`/deleteUser` method is not working as expected: received `{"error":"Order not found"}` in response. So it is always failed in my test. 
I hope sometime Dummy API will be fixed....maybe

File: hw_27/test_api.py
Comment (Stml89) [inline hw_27/test_api.py:1]: Everything in 1 module, this approach does not scale. Lots of code duplication. Need to separate test data, code and tests. But good usage of validation JSON schema

**Review by Stml89 — CHANGES_REQUESTED** [review]

File: hw_27/test_api.py
Comment (ETsiunel) [reply hw_27/test_api.py:1]: reworked tests

**Comment by ETsiunel** [comment]
@Stml89 changes are ready for review

File: .gitignore [hunk 1]
```diff
@@ -16,12 +16,12 @@ venv/
 .DS_Store
 .github/.DS_Store
 .github/workflows/.DS_Store
-
+/Users/yekatsiarynatsyunel/Python/ItAcademyCourse/homeworks/hw_27/config.json
```
Comment (Stml89) [inline .gitignore]:
```suggestion
hw_27/config.json
```

File: hw_27/api_model.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+class BaseAPI:
+    def __init__(self, base_url=BASE_URL):
+        self.base_url = base_url
+        self.auth_token = load_auth_token()
+        self.headers = {
+            "Content-Type": "application/json",
+            "Authorization": f"Bearer {self.auth_token}"
+        }
+
+    def _send_request(self, method, endpoint, data=None, params=None):
```
Comment (Stml89) [inline hw_27/api_model.py:28]: Really nice

**Review by Stml89 — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (ETsiunel) [reply .gitignore]: fixed

**Pull request merged**